src/redux/books/thunkFunction.js

```diff
@@ -0,0 +1,78 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+const CREATE_BOOK = 'bookstore/books/CREATE_BOOK';
+const DELETE_BOOK = 'bookstore/books/DELETE_BOOK';
+const LOAD_BOOK = 'bookstore/books/LOAD_BOOK';
+
+const baseUrl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps';
+const appId = 'k1m6JvTtKocImhvcSEzx';
+
+export const createBook = createAsyncThunk(
+  CREATE_BOOK,
+  async (books) => {
+    await fetch(`${baseUrl}/${appId}/books`, {
+      method: 'POST',
+      body: JSON.stringify(books),
+      headers: {
+        'content-type': 'application/json; charset=UTF-8',
+      },
+    })
+      .then((response) => response.text())
+      .then((data) => data);
+
+    const data = await fetch(`${baseUrl}/${appId}/books`)
+      .then((response) => response.json())
+      .then((data) => data);
+
+    return data;
+  },
+);
+
+export const deleteBook = createAsyncThunk(
```

Comment:
`createAsyncThunk` is used in the project for creating thunk action creators 🐐 Nice!